### PR TITLE
Add `@context` requirement for TMs

### LINKED ISSUE #1455: Requiring `@context` for TMs
Even though it is in the JSON Schema of TMs, we do not say that TMs must have the `@context` field. I think we should add this as an assertion at https://w3c.github.io/wot-thing-description/#thing-model-declaration

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #1455

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1494.html" title="Last updated on May 18, 2022, 3:43 PM UTC (e7d02c1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1494/c45adf5...e7d02c1.html" title="Last updated on May 18, 2022, 3:43 PM UTC (e7d02c1)">Diff</a>

### CODE REVIEW

**Review by sebastiankb — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -6163,6 +6163,9 @@ <h2>Thing Model Declaration</h2>
         <a>Thing Model</a> definitions MUST use the keyword <em>@type</em> at top level 
         and a value of type string or array that equals or respectively contains <code>tm:ThingModel</code>.
     </span>
+    <span class="rfc2119-assertion" id="tm-context-requirement">
+        Additionally, in order to identify it as a JSON-LD document, <a>Thing Model</a> definitions MUST use the keyword <em>@context</em> at top level with same rules as a Thing Description.
```
Comment (danielpeintner) [inline index.html:6167]: Minor: you might want to use `<code>@context</code>` instead of `<em>`

Comment (egekorkan) [reply index.html:6167]: Done. I had copied it from the above one, now both use `<code>`

**Review by thjaeckle — APPROVED** [review]
LGTM 👍

File: index.html [hunk 2]
```diff
@@ -1209,16 +1209,16 @@ <h2>Core Vocabulary Definitions</h2>
                 names are used in a <code>security</code>
                 name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
 <tr class="rfc2119-table-assertion" id="td-vocab-profile--Thing"><td><code>profile</code></td><td>Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation.</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a> or <a>Array</a> of <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-schemaDefinitions--Thing"><td><code>schemaDefinitions</code></td><td>Set of named data schemas.
```
Comment (mlagally) [inline index.html:1212]: unrelated change

File: index.html [hunk 3]
```diff
@@ -1209,16 +1209,16 @@ <h2>Core Vocabulary Definitions</h2>
...
+                To be used in a <code>schema</code>
+                name-value pair inside an 
+                <code>AdditionalExpectedResponse</code> object.</td><td>optional</td><td><a>Map</a> of <a href="#dataschema"><code>DataSchema</code></a></td></tr>
 <tr class="rfc2119-table-assertion" id="td-vocab-uriVariables--Thing"><td><code>uriVariables</code></td><td>Define URI template variables according to [[RFC6570]]  as collection
                 based on DataSchema declarations. The Thing level <code>uriVariables</code> can be used in Thing-level 
                 <code>forms</code> or in Interaction Affordances.
                 The individual variables DataSchema cannot be an ObjectSchema or an ArraySchema.
                 If the same variable is both declared in Thing-level <code>uriVariables</code> and in Interaction Affordance level,
-                 the Interaction Affordance level variable takes precedence.</td><td>optional</td><td><a>Map</a> of <a href="#dataschema"><code>DataSchema</code></a></td></tr>
-<tr class="rfc2119-table-assertion" id="td-vocab-schemaDefinitions--Thing"><td><code>schemaDefinitions</code></td><td>Set of named data schemas.
```
Comment (mlagally) [inline index.html:1218]: how is that related to the issue the PR is trying to solve?

**Review by mlagally — CHANGES_REQUESTED** [review]
This PR contains unrelated changes to the @context requirement.
Thes make reviews more difficult and should be removed.

**Comment by sebastiankb** [comment]
from today's TD call, decided to merge
